types/spectest/tests/consensusdata/proposer/versioned_block_spectest.go

```diff
@@ -0,0 +1,122 @@
+package consensusdataproposer
+
+import (
+	"testing"
+
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/stretchr/testify/require"
+
+	"github.com/bloxapp/ssv-spec/types"
+)
+
+type ProposerSpecTest struct {
+	Name            string
+	Blinded         bool
+	DataCd          []byte
+	DataBlk         []byte
+	ExpectedBlkRoot [32]byte
+	ExpectedCdRoot  [32]byte
+	ExpectedError   string
+}
+
+func (test *ProposerSpecTest) TestName() string {
+	return test.Name
+}
+
+// TODO: make multi proposer spec test for all (upcoming) versions
```

Comment:
Before merging can you open a github issue for this and replace TODO with the issue number?

I don't believe in using TODOs for managing tasks.
I usually use them while I create a PR, and then at the end of the PR I clean them up.

Reply:
```suggestion
ISSUE 219: make multi proposer spec test for all (upcoming) versions
```